Introduction.Rmd

```diff
@@ -125,9 +117,39 @@ The final three chapters describe general best practices not specifically tied t
 
 This is a lot to learn, but don't feel overwhelmed. Start with a minimal subset of useful features (e.g. just an `R/` directory!) and build up over time. To paraphrase the Zen monk Shunryu Suzuki: "Each package is perfect the way it is --- and it can use a little improvement".
 
-## Getting started {#intro-get}
+## devtools, usethis, and you {#intro-usage}
```

Comment:
A significant motivation for this section is to give us a link to our official explanation and wishes. So, yeah, it's a bit of an "article" within the book.

Reply:
I love this title!